resources/lib/controller.py

```diff
@@ -263,47 +280,124 @@ def show_history(args, api: API):
                        "mode": args.mode},
                       is_folder=True)
 
-    view.end_of_directory(args)
+    view.end_of_directory(args, "episodes")
 
     return True
 
 
-def list_seasons(args, mode, api: API):
-    """ view all available anime seasons and filter by selected season
+def show_resume_episodes(args, api: API):
+    """ shows episode to resume for watching animes
     """
-    season_filter: str = getattr(args, "season_filter", "")
+    items_per_page = 20
```

Comment:
I'd bump that up to 50 to be in line with the other views. Also, if you limit it, you'd need a pagination - not sure if there are that many items, though.